### DISCUSSION TITLE
Error when submitting to public leaderboard

### DISCUSSION THREAD

**Discussion by vigsivan** [facebookresearch/fastMRI#240] [General]
Hello,

When submitting the URL to my tar.gz file, I get the following error message:

`UnboundLocalError: local variable 'request' referenced before assignment`

The reconstructions are hosted on Google Drive. Do you know what the issue might be?

Please let me know if there is additional information that I can provide.

Thanks!

**Comment by mmuckley** [comment]
Hello @vigsivan, please see Issue #223. Google Drive made some API changes that broke our site, but I haven't had the time lately to try to figure out a way to update the website. For now, you'll need to use another storage service.

**Comment by vigsivan** [comment]
Okay, I see. Thanks!
